style.css

```diff
@@ -284,6 +285,7 @@ body {
 
 .content button:hover {
   background-color: #396df2;
+  color: #fff;
 }
 
 @media (max-width: 500px) {
```

Comment:
- [ ] According to the project requirements. You should  `Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).`  That is: 
```
    @media (min-width: 768px) {
       // Your css code here.....
     }
```

[your-mobile-responsive.webm](https://user-images.githubusercontent.com/26597432/212854055-ff5674e0-ced5-45f5-9501-f816d0c3523a.webm)